oak-segment-azure/pom.xml

```diff
@@ -71,18 +73,28 @@
                             org.apache.jackrabbit.oak.segment.azure,
                             org.apache.jackrabbit.oak.segment.azure.queue,
                             org.apache.jackrabbit.oak.segment.azure.util,
+                            com.fasterxml.jackson.dataformat.xml,
+                            com.fasterxml.jackson.dataformat.xml.deser,
                             com.microsoft.azure.storage,
                             com.microsoft.azure.storage.core,
                             com.microsoft.azure.storage.blob,
                             com.azure.core.credential,
-                            com.azure.identity
+                            com.azure.identity,
+                            com.azure.blob,
+                            com.azure.storage.common,
+                            com.azure.storage.internal.avro.implementation
                         </Export-Package>
                         <Embed-Dependency>
                             azure-storage,
                             azure-keyvault-core,
                             azure-core,
                             azure-identity,
                             azure-json,
+                            azure-xml,
```

Comment:
Arer we embedding the new SDK? Why?

(We embedded the old one due to the Guava dependency issue, but this should not be needed here)

Reply:
I tried without and I got the following error at build:
`[ERROR] Bundle oak-segment-azure:1.71-ierandra-T20241104112150-3ccdb109e0 is importing package(s) com.azure.xml in start level 15 but no bundle is exporting these for that start level.`

Reply:
In the oak-it-osgi tests? In which case we may have to add the dependency there.

Reply:
no, when building CQ: https://jenkins-evergreen.ci.corp.adobe.com/job/Sprouts/job/aemcs-developer-releases/5335/artifact/1-BuildAEM.log